docs/examples/physics_using_pybullet/viz_domino.py

```diff
@@ -0,0 +1,238 @@
+# The following imports are necessary for physics simulations
+import numpy as np
+from fury import window, actor, ui, utils
+import itertools
+import pybullet as p
+
+# Instantiate Pybullet client.
+client = p.connect(p.DIRECT)
+
```

Comment:
If you are not sure whats going on then you can do `client = p.connect(p.GUI)` to understand render in pybullet world.